### PR TITLE
Fixes #7635: syslog-ng's loghost regex does not match with set line

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7635

### CODE REVIEW

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -753,9 +757,14 @@ bundle edit_line edit_syslog_conf_file(line_to_add, pattern_to_remove)
 
   delete_lines:
       "${pattern_to_remove}";
+
+      # Remove any unexpanded variables that may have slipped through due to old bugs or race conditions at installation time
       "\$\(syslog_ng_conf\)";
       "\$\{syslog_ng_conf\}";
 
+      # Remove the syslog config when the loghost was just called "loghost" (now called "rudder_loghost")
+      "filter\ f\_local\_rudder\{facility\(local6\)\ and\ program\(\"rudder\.\*\"\)\;\}\;destination\ loghost\ \{(tcp|udp)\(\"[^\"]+\".*";
```
Comment (peckpeck) [inline techniques/system/common/1.0/promises.st]: Why is there backslash in front of spaces and underscores ?

**Comment by peckpeck** [comment]
Aside of that, everything is good

File: techniques/system/common/1.0/promises.st (hunk 1 above)
Comment (jooooooon) [inline techniques/system/common/1.0/promises.st]: Dunno. It's copy pasted from the regex we used above and we know works...

**Comment by jooooooon** [comment]
PR updated

**Comment by peckpeck** [comment]
OK, merging

**Pull request merged**